### ISSUE TITLE
Pawns prefer to consume kibble to coconuts

### ISSUE THREAD

**Issue by l29ah** [skyarkhangel/Hardcore-SK#1932]
WHAT

**Comment by sidfu** [comment]
not a bug. pawns will grab any food they allowed that is close. base game has a  food setup u can use. also for questions ask in discord instead of opening new issues.

**Comment by l29ah** [comment]
No, in the base game a pawn will never eat kibble if more sensible food is available, while in Hardcore-SK i need to explicitly disable it.

**Comment by zakhad** [comment]
Just use WM smarter food selection mod :S and use the various patches to it in #patches channel of the hcsk discord. Which will sort out your issue your having with food prioritization.

It goes doubly so if your using seeds please because your animals by default don't care about whose potato plant they're eating, with smarter food they will always prioritise various non beneficial plants like moss or decorative flowers as the equivalent of grass.

**Comment by Voker57** [comment]
fixed by linked MR
